test/e2e/persistent_volumes.go

```diff
@@ -522,6 +515,14 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 				nfsServerPod = startVolumeServer(c, NFSconfig)
 				serverIP = nfsServerPod.Status.PodIP
 				framework.Logf("NFS server IP address: %v", serverIP)
+			} else {
+				// If the nfs server does exist, check that it is Running and the volume is exported.
```

Comment:
These are good tests. See also pr [38214](https://github.com/kubernetes/kubernetes/pull/38214/files#diff-c83ebba11ffd12bd594ac8aa2f9301ff) where it is no longer an error if the server pod already exists.